build-databroker-cli.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# Building all currently supported targets for databroker-cli.
+# Uses cross for cross-compiling. Needs to be executed
+# before docker build, as docker collects the artifacts
+# created by this script
+# this needs the have cross, cargo-license and kuksa sbom helper
+# installed
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This script will build databroker-cli for different architectures uing cross
+# (https://github.com/cross-rs/cross)
+# Artifacts will be out in the directory ./dist in the form required for the
+# Dockerfile-cli
+#
+# You run it like
+#
+# ./build-databroker-cli.sh plattforms
+#
+# where platform can be one or more of
+#
+# arm64, amd64, riscv64, i.e. the following are valid commandlines
+#
+# ./build-databroker-cli.sh amd64
+# ./build-databroker-cli.sh amd64 arm64 riscv64
+#
+# you can enable features that will be passed to cargo
+# by setting the environment variable KUKSA_DATABROKERCLI_FEATURES, i.e.
```

Comment:
We miss the example feature line:

`# KUKSA_DATABROKERCLI_FEATURES=databroker/viss,databroker/tls`

(But do we need/use any features for CLI? See other comment for features used in CI, we use VISS only for databroker, not databroker-cli, right?)

Reply:
I did not add an example, becasue those I think are not valid CLI features, and I dit not know, what were

Reply:
Then just remove the `, i.e.`

Reply:
done